src/environments/gridworlds.jl

```diff
@@ -0,0 +1,26 @@
+export RLBaseGridWorld
+import .GridWorlds
+
+mutable struct RLBaseGridWorld{T<:GridWorlds.AbstractGridWorld} <: RLBase.AbstractEnv
+    env::T
+end
+
+Base.convert(::Type{RLBase.AbstractEnv}, env::GridWorlds.AbstractGridWorld) = convert(RLBaseGridWorld, env)
```

Comment:
This is a useful converter. Could you also add a constructor directly, like `GridWorldEnv(;state_size=(7,7), game="EmptyGridWorld", kwargs...)`